sdk/tests/deqp/functional/gles3/builtinprecision01.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
```

Comment:
I didn't notice this before but can this doctype either be removed, or simplified to "DOCTYPE html"?